### DISCUSSION TITLE
关于ddns-go隐藏cmd窗口，并后台运行的方法

### DISCUSSION THREAD

**Discussion by Ren9527** [jeessy2/ddns-go#524] [Ideas]
ddns-go目录下，创建VBS脚本，内容如下：
Set ws = CreateObject("Wscript.Shell")  
ws.run "ddns-go.exe",vbhide
并创建快捷方式添加到启动目录（本方案来源于Alist的启动方法）

**Comment by fyhbauyg** [comment]
> set ws = CreateObject（“Wscript.Shell”） ws.run “ddns-go.exe”，vbhide 并创建快捷方式添加到启动目录（本方案来源于Alist的启动方法）

应该把全部符号换成英文否则会报错
set ws=CreateObject("Wscript.Shell")
ws.run "ddns-go.exe",vbhide

**Comment by zymywz** [comment]
[ddns-go.vbs.txt](https://github.com/user-attachments/files/23443601/ddns-go.vbs.txt)
下载ddns-go.vbs.txt,重命名去掉后缀.txt,剪切至ddns-go目录下，并创建快捷方式添加到启动目录（本方案来源于Alist的启动方法）

**Comment by Traveler7274** [comment]
我使用该脚本时添加到开机自启动后，报错代码80070002，系统无法找到指定得文件，我不确定是快捷方式的问题，还是因为win11，总之脚本运行目录不在脚本所在文件夹。如果遇到了相同的问题，可以尝试以下修改，按照提示填写路径即可。

> set ws=CreateObject("Wscript.Shell")
   ws.currentdirectory = "替换为脚本文件和ddns-go.exe所在的绝对路径"
   ws.run` "ddns-go.exe",vbhide

或下载以下文件，修改好路径后，将.txt删掉
[ddns-go.vbs.txt](https://github.com/user-attachments/files/24102882/ddns-go.vbs.txt)

**Reply by zymywz** [reply]
是否重命名脚本。
